pyproject.toml

```diff
@@ -1,4 +1,4 @@
-# Copyright 2022 Canonical Ltd.
+# Copyright 2021 Canonical Ltd.
```

Comment:
Why change 2022 for 2021? :laughing:

Reply:
I always wanted to try time traveling :D